docs/Solidity.g4

```diff
@@ -92,6 +93,9 @@ modifierInvocation
 eventDefinition
   : 'event' identifier eventParameterList AnonymousKeyword? ';' ;
 
+errorDefinition
+  : 'error' identifier eventParameterList AnonymousKeyword? ';' ;
```

Comment:
Why anonymous?

Reply:
oh that was a bad thing! (copy'n'paste blindfolded) :)